pyronear/models/ssresnet18.py

```diff
@@ -51,9 +51,10 @@ def forward(self, x):
             s = i % self.frame_per_seq
             x2[i // self.frame_per_seq, s * x.shape[1]:(s + 1) * x.shape[1], :, :] = x[i, :, :, :]
 
-        x = x2
         if x.device.type.startswith('cuda'):
-            x = x.cuda()
+            x = x2.to(x.device)
+        else:
+            x = x2
```

Comment:
You can shorten that to 
`x = x2.to(x.device)` since the device can be a CPU as well ;)

Reply:
indeed :)